src/libmongoc/src/mongoc/mongoc-topology.c

```diff
@@ -1732,3 +1558,46 @@ _mongoc_topology_bypass_cooldown (mongoc_topology_t *topology)
    BSON_ASSERT (topology->single_threaded);
    topology->scanner->bypass_cooldown = true;
 }
+
+/* Called from application threads
+ * Caller must hold topology lock.
+ * Locks topology description mutex to copy out server description errors.
+ * CLEANUP: this has nothing to do with background monitoring. Move to
```

Comment:
Adding a note here: did you want to clean this up in this PR or in a follow-up?

Reply:
Ah, I moved it, but forgot to update the comment. Fixed.